src/Microsoft.PowerShell.Commands.Utility/commands/utility/group-object.cs

```diff
@@ -246,53 +248,54 @@ public SwitchParameter NoElement
         internal static void DoGrouping(OrderByPropertyEntry currentObjectEntry, bool noElement, List<GroupInfo> groups, Dictionary<object, GroupInfo> groupInfoDictionary,
             OrderByPropertyComparer orderByPropertyComparer)
         {
-            if (currentObjectEntry != null && currentObjectEntry.orderValues != null && currentObjectEntry.orderValues.Count > 0)
+            var currentObjectorderValues = currentObjectEntry.orderValues;
```

Comment:
Should it be `currentObjectOrderValues`?

Reply:
yes

Reply:
I don't see the fix.

Reply:
Please address the comment and fix case.